docs/debugging-install.md

```diff
@@ -0,0 +1,59 @@
+# Debugging Installation Issues
+
+## General
+
+#### `invalid configuration: no configuration has been provided`
+This typically means that no `kubeconfig` file can be found for the Ark client to use. Ark looks for a kubeconfig in the 
+following locations:
+* the path specified by the `--kubeconfig` flag, if any
+* the path specified by the `$KUBECONFIG` environment variable, if any
+* `~/.kube/config`
+
+#### Backups or restores stuck in `New` phase
+This implies that the Ark server is not up and running. Check the pod description and logs for errors:
+```
+kubectl -n heptio-ark describe pods
+kubectl -n heptio-ark logs deployment/ark
+```
+
+## AWS
+#### `NoCredentialProviders: no valid providers in chain`
```

Comment:
Heading plus code formatting renders a bit unexpectedly. Can we describe more generically for the heading string and then put the actual error string in an intro para?  (Same for similar below)

Reply:
I wanted to make it symptom-driven (if I was a user with an issue, I'd search by error). Maybe I structure each one with a generic header, then have a "symptoms" and "resolution" sub-sectio for each?

Reply:
Eh, I tried that and it adds a lot of noise. Personally I prefer it as-is - do you feel strongly about this?

Reply:
nope, if you tried other things and this seems like best UX, it's all good.